python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -245,10 +268,26 @@ def makePlot(
         ax.tick_params(labelsize=7)
 
         ax.set_aspect("equal")
+
+        if self.addHistogram:
+            bins = np.linspace(
+                (self.plotMin if (self.plotMin is not None) else min(data["z"].astype(float))),
+                (self.plotMax if (self.plotMax is not None) else max(data["z"].astype(float))),
+                self.histBins,
+            )
+            histAx.hist(data["z"].astype(float), bins=bins)
+            histAx.set_xlabel(self.zAxisLabel)
+            histAx.set_ylabel("Bin count")
+            underflow = np.count_nonzero(data["z"] < bins[0])
```

Comment:
Consider adding counts of non-finite and/or nan values if there are likely to be any.

Reply:
Added "Non-finite = ..."